clang/test/SemaCXX/lambda-expressions.cpp

```diff
@@ -598,7 +598,7 @@ struct S1 {
 };
 
 void foo1() {
-  auto s0 = S1([name=]() {}); // expected-error {{expected expression}}
+  auto s0 = S1([]() {}); // Remove invalid capture, no diagnostic expected
```

Comment:
why change the test?  this one seemed like a useful diagnostic, and one I'd like to see.

Reply:
@charan-003